packages/ui/components/dialog/src/LionDialog.js

```diff
@@ -1,23 +1,51 @@
-import { html, LitElement } from 'lit';
-import { OverlayMixin, withModalDialogConfig } from '@lion/ui/overlays.js';
-
-export class LionDialog extends OverlayMixin(LitElement) {
-  /**
-   * @protected
-   */
-  // eslint-disable-next-line class-methods-use-this
-  _defineOverlayConfig() {
-    return {
-      ...withModalDialogConfig(),
-    };
+import { css, html, LitElement } from 'lit';
+
+export class LionDialog extends LitElement {
+  static properties = {
+    opened: {},
+  };
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('close', this.#close);
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.removeEventListener('close', this.#close);
+  }
+
+  get #dialog() {
+    return this.shadowRoot.querySelector('dialog');
   }
 
+  #open() {
+    this.#dialog.showModal();
+    this.opened = true;
+  }
+
+  #close() {
+    this.#dialog.close();
+    this.opened = false;
+  }
+
+  static styles = css`
+    :host {
+      --border: 1px solid black;
+    }
+    dialog {
+      border: var(--border);
+    }
+  `;
+
   render() {
     return html`
-      <slot name="invoker"></slot>
-      <div id="overlay-content-node-wrapper">
+      <button part="invoker" @click=${this.#open}>
+        <slot name="invoker"></slot>
+      </button>
+      <dialog id="overlay-content-node-wrapper">
```

Comment:
I don't think we need this anymore.

```suggestion
      <dialog>
```